Xamarin.Forms.Platform.Android/FastRenderers/VisualElementRenderer.cs

```diff
@@ -88,6 +81,8 @@ void Dispose(bool disposing)
 
 		void OnElementChanged(object sender, VisualElementChangedEventArgs e)
 		{
+			var reference = Guid.NewGuid().ToString();
```

Comment:
Nope

Reply:
I think there might be a fourth one of these somewhere, but my "nope" is missing.